tailwind.config.js

```diff
@@ -1,11 +1,21 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-  content: [
-    "./index.html",
-    "./src/**/*.{js,ts,jsx,tsx}",
-  ],
+  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  important: '#root',
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
+        logo: "url('src/assets/images/logo.png')",
+      },
+      colors: {
+        primary: '#ff0000',
+        secondary: '#00ff00',
+        tertiary: '#0000ff',
```

Comment:
Ville nok definere vores baggrundsfarve mm. her. Primary, secondary og tertiary bliver ikke brugt nogen steder så vidt jeg kan se.
```suggestion
orange: #FFCA80,
green: #D6FBE4,
```